src/loader/mod.rs

```diff
@@ -12,19 +12,20 @@
 //! - [KernelLoaderResult](struct.KernelLoaderResult.html): the structure which loader
 //! returns to VMM to assist zero page construction and boot environment setup
 //! - [Elf](struct.Elf.html): elf image loader
+//! - [Arm64Pe](struct.Arm64Pe.html): arm64_pe image loader
 //! - [BzImage](struct.BzImage.html): bzImage loader
 
 extern crate vm_memory;
 
 use std::error::{self, Error as KernelLoaderError};
 use std::ffi::CStr;
 use std::fmt::{self, Display};
-#[cfg(any(feature = "elf", feature = "bzimage"))]
-#[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
+#[cfg(any(feature = "elf", feature = "bzimage", feature = "arm64_pe"))]
+#[cfg(any(target_arch = "x86", target_arch = "x86_64", target_arch = "aarch64"))]
```

Comment:
These 2 lines cover all the features and all the supported architectures, you could remove them to keep the code simple